src/transformers/generation_utils.py

```diff
@@ -526,6 +529,18 @@ def _generate_no_beam_search(
         sent_lengths = input_ids.new(batch_size).fill_(max_length)
 
         past = None
+        past2 = None
+
+        # Initialize MMI antiLM settings
+        if antilm_params is None:
```

Comment:
This is `None` by default, right? So by default this branch would be hit and `generate()` would `quit()`?

Reply:
That line is just error-handling for my research code.